src/it/scala/org/zalando/spearheads/innkeeper/AcceptanceSpecsHelper.scala

```diff
@@ -58,31 +60,51 @@ object AcceptanceSpecsHelper extends ScalaFutures {
     futureResponse.futureValue
   }
 
-  def getSlashRoutes(token: String): HttpResponse = {
+  def getSlashRoutes(token: String = ""): HttpResponse = {
     val futureResponse = Http().singleRequest(HttpRequest(uri = routesUri,
-      headers = Seq[HttpHeader](Authorization(OAuth2BearerToken(token)))))
+      headers = headersForToken(token)))
     futureResponse.futureValue
   }
 
   def getSlashRoutesByName(token: String, name: String): HttpResponse = {
     val futureResponse = Http().singleRequest(HttpRequest(uri = routeByNameUri(name),
-      headers = Seq[HttpHeader](Authorization(OAuth2BearerToken(token)))))
+      headers = headersForToken(token)))
     futureResponse.futureValue
   }
 
-  def getSlashRoute(token: String, id: Long): HttpResponse = {
-    slashRoute(token, id)
+  def getSlashRoute(id: Long, token: String = ""): HttpResponse = {
+    slashRoute(id, token)
   }
 
-  def deleteSlashRoute(token: String, id: Long): HttpResponse = {
-    slashRoute(token, id, HttpMethods.DELETE)
+  def deleteSlashRoute(id: Long, token: String = ""): HttpResponse = {
+    slashRoute(id, token, HttpMethods.DELETE)
   }
 
-  private def slashRoute(token: String, id: Long, method: HttpMethod = HttpMethods.GET): HttpResponse = {
-    val futureResponse = Http().singleRequest(HttpRequest(uri = routeUri(id), method = method,
-      headers = Seq[HttpHeader](Authorization(OAuth2BearerToken(token)))))
+  private def slashRoute(id: Long, token: Option[String] = None, method: HttpMethod = HttpMethods.GET): HttpResponse = {
+    val futureResponse = Http().singleRequest(
+      HttpRequest(
+        uri = routeUri(id),
+        method = method,
+        headers = headersForToken(token)
+      )
+    )
     futureResponse.futureValue
   }
+
+  private def headersForToken(token: Option[String]): Seq[HttpHeader] = {
+    val headers = token match {
+      case Some(token) => Seq[HttpHeader](Authorization(OAuth2BearerToken(token)))
+      case None => Seq()
+    }
+    headers
+  }
+
+  implicit def stringToOption(string: String): Option[String] = {
```

Comment:
I don't want to use this kind of implicit conversions in the production code... I'll keep it into the tests...